src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/ActiveCampaignContactsWorkflow.cs

```diff
@@ -67,6 +67,13 @@ public override WorkflowExecutionStatus Execute(WorkflowExecutionContext context
                 // Check if contact exists.
                 var contacts = _contactService.Get(email).ConfigureAwait(false).GetAwaiter().GetResult();
 
+                if(contacts.Contacts.Count > 0 && !_settings.AllowContactUpdate)
+                {
+                    _logger.LogError("ActiveCampaign contact update is not allowed.");
```

Comment:
```suggestion
                    _logger.LogInformation("Contact already exists in ActiveCampaign and workflow is configured to not apply updates, so update of information was skipped.");
```

Just think we shouldn't log an error here, as it's normal behaviour.